packages/chaire-lib-backend/src/config/server.config.ts

```diff
@@ -405,6 +405,19 @@ export const setProjectConfiguration = (newConfig: Partial<ProjectConfiguration<
         }
     }
 
+    // If there are still section configurations in the defaultPreferences section of the project config, set it as section config.
+    // FIXME Remove this if a reasonable time after march 2025, to give time to instances to update their configuration
+    const defaultPreferences = (newConfig as any)?.defaultPreferences;
+    if (defaultPreferences && defaultPreferences.sections) {
```

Comment:
Can we have a console.warn() here that says that we are using a deprecated feature. 
(To help make sure we have migrated everything)

Reply:
done